packages/userstorage/src/browser/user-storage-service-filesystem.ts

```diff
@@ -21,8 +21,7 @@ import { FileSystemWatcher, FileChangeEvent } from '@theia/filesystem/lib/browse
 import { FileSystem } from '@theia/filesystem/lib/common';
 import URI from '@theia/core/lib/common/uri';
 import { UserStorageUri } from './user-storage-uri';
-
-export const THEIA_USER_STORAGE_FOLDER = '.theia';
+import { THEIA_USER_STORAGE_FOLDER } from '../common/constants';
```

Comment:
I wonder whether we can expose `UserStorageServiceFilesystemImpl.userStorageFolder` instead and use it in the plugin system?

Reply:
As for me it is not worth it. We've already use constant which is used to obtain userStorageFolder via the same underlying backend service.

Reply:
How are you going to change the constant? It is not even in DI context to be redound once. This PR moves the hardcoded value to another place. It does not solve the issue.

Reply:
Answered [here](https://github.com/eclipse-theia/theia/pull/6650#issuecomment-559790390).